quantized_mesh_encoder/encode.py

```diff
@@ -3,14 +3,14 @@
 import numpy as np
 
 from .bounding_sphere import bounding_sphere
-from .constants import HEADER, NP_STRUCT_TYPES, VERTEX_DATA
+from .constants import HEADER, NP_STRUCT_TYPES, VERTEX_DATA, EXTENSION_HEADER
```

Comment:
I haven't set up CI linting on this repo, but could you run isort? It would basically put these imported symbols in alphabetical order, and put the `.normals` import line before `.occlusion` also in alphabetical order.